recipes/glib/all/conanfile.py

```diff
@@ -53,20 +52,22 @@ def config_options(self):
             del self.options.with_selinux
 
     def build_requirements(self):
-        self.build_requires("meson/0.54.2")
+        self.build_requires("meson/0.55.1")
         if self.settings.os == "Windows":
             self.build_requires("pkgconf/1.7.3")
 
     def requirements(self):
+        self.requires("zlib/1.2.11")
+        self.requires("libffi/3.3")
         if self.options.with_pcre:
-            self.requires("pcre/8.41")
+            self.requires("pcre/8.44")
         if self.options.with_elf:
             self.requires("libelf/0.8.13")
         if self.settings.os == "Linux":
             if self.options.with_mount:
                 self.requires("libmount/2.33.1")
             if self.options.with_selinux:
-                self.requires("libselinux/2.9")
+                self.requires("libselinux/3.1")
         else:
             # for Linux, gettext is provided by libc
             self.requires("libgettext/0.20.1")
```

Comment:
Can libgettext also be updated ? I believe libgettext has been depreciated for gettext

Reply:
not until until https://github.com/conan-io/conan-center-index/pull/1424 is merged

Reply:
my bad saw the depreciation notice didn't know the package was pending.